skbio/stats/ordination/_ordination_results.py

```diff
@@ -399,3 +399,40 @@ def _format_attribute(self, attr, attr_label, formatter):
         else:
             formatted_attr = formatter(attr)
         return "\t%s: %s" % (attr_label, formatted_attr)
+
+    def rename(self, id_type, rename_dict, strict=True):
```

Comment:
Can `id_type` be described in the Parameters block?

Reply:
Yes, I'll take care of that!